### ISSUE TITLE
TOC missing on Documentation

### ISSUE THREAD

**Issue by ComedyTomedy** [phillord/lentic#56]
Documentation on [Phil's homepage at Newcastle Uni](http://homepages.cs.ncl.ac.uk/phillip.lord/lentic/lenticular.html) is missing the table of contents, except the Caveat. It took me a minute to realise the rest of the docs were there at all.

Hopefully this is an easy fix ☺ given that it says the documentation generation isn't perfect, it's quite funny really.

**Comment by phillord** [comment]
Yeah. It's actually correct, but it's not very clean at all. I will see what I can do.

**Comment by ComedyTomedy** [comment]
Don't worry, doesn't need to be a priority, I was just making a note in case it was useful to you.

I've since noticed m-buffer-el is similar, so I guess it's just part of the current state of lentic generated documentation :)
